pkgs/development/python-modules/cppy/default.nix

```diff
@@ -6,11 +6,11 @@
 
 buildPythonPackage rec {
   pname = "cppy";
-  version = "1.1.0";
+  version = "1.2.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "4eda6f1952054a270f32dc11df7c5e24b259a09fddf7bfaa5f33df9fb4a29642";
+    sha256 = "sha256-leiGLk+CbD8qa3tlgzOxYvgMvp+UOqDQp6ay74UK7/w=";
   };
 
   # Headers-only library, no tests
```

Comment:
Well, it has a pretty basic one https://github.com/nucleic/cppy/tree/main/tests and it should have a pythonImportsCheck.